app/controllers/admin/resources_controller.rb

```diff
@@ -48,5 +56,24 @@ def destroy
     def resource_params
       params.require(:resource).permit(:name, :description, :quantity, :used, :conference_id)
     end
+
+    def check_successful_update(increment_used_resource_flag)
```

Comment:
Are you suggesting to carry out update this way 
`@resource.update(name: params[:resource][:name],
                            description: params[:resource][:description],
                            used: params[:resource][:used].to_i + params[:used_count].to_i,
                            quantity: params[:resource][:quantity]
                           )`
Or something else?